src/web/src/alert_list.ecpp

```diff
@@ -126,8 +128,8 @@ bool database_ready;
         else
             if (!is_requeststate (state.c_str ())) {
                 log_error ("state = '%s' is not a valid alert state.", state.c_str ());
-                http_die ("request-param-bad", "state", std::string ("value '").append (state).append ("'").c_str (),
-                        std::string ("one of the following values [ ALL | ALL-ACTIVE | ACTIVE | ACK-WIP | ACK-IGNORE | ACK-PAUSE | ACK-SILENCE | RESOLVED ]").c_str ());
+                http_die ("request-param-bad", "state", std::string (TRANSLATE_ME ("value '")).append (state).append ("'").c_str (),
```

Comment:
I think it would be better to use char *chs = zsys_sprintf("value '%s'", state.c_str());
std::string s(chs); zsys_free(chs); instead as it gives more details about what's going on, instead of current string that translators will see (with the apostrophe without much content given): value '

Reply:
i don't understand what you mean...